<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiMainClass</class>
 <widget class="QMainWindow" name="guiMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>guiMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="midiDeviceList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deviceScan">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SCAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="midiMessageTest">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>322</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="controllerMappingForm">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Root Note</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="yellowButtonValue"/>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="blueButtonValue"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="rootNote"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="greenButtonValue"/>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="orangeButtonValue"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="redButtonValue"/>
     </item>
     <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Controller Mapping</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="greenOff_Label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>102</width>
      <height>163</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/GREEN_NOT_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="greenOn_Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>102</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/GREEN_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="redOn_Label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/RED_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="yellowOff_Label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/YELLOW_NOT_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="yellowOn_Label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/YELLOW_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="blueOff_Label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/BLUE_NOT_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="blueOn_Label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>570</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/BLUE_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="orangeOff_Label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/ORANGE_NOT_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="orangeOn_Label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/ORANGE_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="redOff_Label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="guiMain.qrc">:/Buttons/RED_NOT_PRESSED.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="controllerList">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Controller 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="controller2_Radio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Controller 3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="controller3_Radio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Controller 4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="controller4_Radio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Controller 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="controller1_Radio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>redOff_Label</zorder>
   <zorder>redOn_Label</zorder>
   <zorder>greenOn_Label</zorder>
   <zorder>greenOff_Label</zorder>
   <zorder>midiDeviceList</zorder>
   <zorder>deviceScan</zorder>
   <zorder>midiMessageTest</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>yellowOff_Label</zorder>
   <zorder>yellowOn_Label</zorder>
   <zorder>blueOff_Label</zorder>
   <zorder>blueOn_Label</zorder>
   <zorder>orangeOff_Label</zorder>
   <zorder>orangeOn_Label</zorder>
   <zorder>formLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="guiMain.qrc"/>
 </resources>
 <connections/>
</ui>
